src/test/java/net/minecraftforge/debug/EnumValuesInlineTest.java

```diff
@@ -0,0 +1,56 @@
+package net.minecraftforge.debug;
+
+import net.minecraftforge.common.util.EnumHelper;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+import org.apache.logging.log4j.LogManager;
+
+import javax.annotation.Nullable;
+
+/**
+ * A test for {@link net.minecraftforge.fml.common.asm.transformers.EnumConstructorTransformer}.
+ * Also tests that the JVM does <i>not</i> inline the Enum.$VALUES field.
+ */
+@SuppressWarnings("unused")
+@Mod(modid = EnumValuesInlineTest.MOD_ID, name = "Enum Values Transformer Test", version = "0.1")
+public class EnumValuesInlineTest
```

Comment:
If you let that test run under launchclassloader, you can modify this one too. see the updated version here:
https://gist.github.com/liach/cdde4a79e8b493f76c34f43f50d331ba

Reply:
This is a test mod, it always runs under LCL

Reply:
You should move it into the test package so that it is no longer a test mod (people may live easier in this case, too)